File: Server/CQS/DTOs/SpecialInspectionDto.cs
using System;

namespace Server.CQS.DTOs
{
    public class SpecialInspectionDto
    {
        public Guid Id { get; set; }
        public string CreatedBy { get; set; }
        public long CreatedDate { get; set; }
        public string LastModifiedBy { get; set; }
        public long LastModifiedDate { get; set; }
        public bool IsActive { get; set; }
        public Guid? UserId { get; set; }
        public Guid ColonyId { get; set; }
        public string[] Harvest { get; set; }
        public string[] Feeds { get; set; }
        public string[] Treatments { get; set; }
        public string[] TreatmentDetails { get; set; }
        public string[] Wintering { get; set; }
        public string Growth { get; set; }
    }
}

File: App/app/src/main/java/com/example/apiverte_colony_management/DTOs/SpecialInspectionDTO.java
package com.example.apiverte_colony_management.DTOs;

import java.time.LocalDateTime;
import java.util.UUID;

public class SpecialInspectionDTO {
    public UUID Id;

    public String Name;

    public Boolean IsActive;

    public String CreatedBy;

    public long CreatedDate;
    public String LastModifiedBy;
    public long LastModifiedDate;

    public UUID UserId;
    public UUID ColonyId;
    public String[] Harvest;
    public String[] Feeds;
    public String[] Treatments;
    public String[] TreatmentDetails;
    public String[] Wintering;
    public String Growth;
}
